### PR TITLE
added new changes

### PR DESCRIPTION
**Pull request by evanhearne**
@denispop9 can you approve these changes? They're nothing much, just whatever that was done an hour ago. Thanks :)

### CODE REVIEW

**Comment by evanhearne** [comment]
@laurafitzgerald

File: calculatorproject.js [hunk 1]
```diff
@@ -1 +1,5 @@
-calculatorproject.js
\ No newline at end of file
+function addNumberToDisplay(number) {
+  // Function Implementation...
+}
+
+displayScreen.value = number;
```
Comment (laurafitzgerald) [inline calculatorproject.js]: @evanhearne the function is defined well here -> this line displayScreen.value = number; is outside the function however so wont be called. The commented line `// Function Implementation...` is there to guide you where to put the code that you want executed when the function is called.

**Review by laurafitzgerald — CHANGES_REQUESTED** [review]

File: calculatorproject.js (hunk 1 above)
Comment (evanhearne) [reply calculatorproject.js]: OK, thank you. Will make changes now.

**Comment by evanhearne** [comment]
@laurafitzgerald added new changes. Thanks a lot for your help, and patience! haha :)

**Review by laurafitzgerald — APPROVED** [review]
@evanhearne tested the changes locally and it works as expected. Feel free to merge this when you have time.

**Comment by laurafitzgerald** [comment]
@evanhearne just revisiting a couple of prs. I would suggest a more descriptive commit message. This gets easier in time and can be a tricky at first but it does help to convey to others and indeed to yourself what exactly it is that you are adding via the commit.

**Pull request merged**